core/config/docs/chains-solana.toml

```diff
@@ -40,6 +40,10 @@ BlockHistoryPollPeriod = '5s' # Default
 ComputeUnitLimitDefault = 200_000 # Default
 # EstimateComputeUnitLimit enables or disables compute unit limit estimations per transaction. If estimations return 0 used compute, the ComputeUnitLimitDefault value is used, if set.
 EstimateComputeUnitLimit = false # Default
+# BlockHistorySize is the number of blocks to take into consideration when using FeeEstimatorMode = 'blockhistory' to determine compute unit price.
+# If set to 1, LatestBlockEstimator will be used. The compute unit price will be determined by the median of the last block's compute unit prices. 
+# If set N > 1, MultipleBlocksEstimator will be used. The compute unit price will be determined by the average of the medians of the last N blocks' compute unit prices.
```

Comment:
I think giving each of these scenarios a name like this might confuse people since they're both still the Block History Estimator. Think you can just skip to the explanation here
```suggestion
# If set to 1, the compute unit price will be determined by the median of the last block's compute unit prices. 
# If set N > 1, the compute unit price will be determined by the average of the medians of the last N blocks' compute unit prices.
```

Reply:
Also for the `N > 1` condition, it might be worth adding a warning or disclaimer at the end essentially mentioning that the higher the number the more RPC calls the estimator will need to make. Just so someone configuring this is conscious of the load on their Solana node and not just assuming we can minimize this with batching like in EVM.